docs/topics/configmap-mission-intro.adoc

```diff
@@ -8,7 +8,7 @@ This mission shows you how to:
 .About Externalized Configuration
 It is important for the application configuration to be externalized and separated from its code. This allows the application configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your code base and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
 Microservices architectures and polyglot environments add a layer of complexity to managing an applications configuration. Applications are comprised of independent services and the configuration is distributed.
-This creates a challenge of keeping the configuration synchronized and accessible from all services.
+This creates the challenge of keeping the configuration synchronized and accessible from all services.
```

Comment:
This creates a maintenance challenge to keep all configuration synchronized and accessible from all services needing it.